controllers/users.js

```diff
@@ -1,22 +1,21 @@
-var express = require('express'),
-    router = express.Router(),
-    User = require('../models/user');
+var express = require('express');
+var router = express.Router();
+var User = require('../models/user');
 
-
-// Get a User
+// Get a user by id
 router.get('/:id', function(req, res) {
   User.get(req.params.id).then(function(user) {
-    res.json({user: user});
+    res.status(200).json({user: user});
   }).catch(function(error) {
-    res.status(500).json({error: error}); 
+    res.status(500).json({error: error});
   });
 });
 
-// Create a new User.
+// Create a new user
 router.post('/', function(req, res) {
   if (req.body.hasOwnProperty('email') && req.body.hasOwnProperty('password')) {
```

Comment:
So I've had weird issues with trying to return early in express routes in the past (when working on Boozebot api -- it basically kept freaking out and claiming that I was setting the res status twice when I wasn't because I was returning), which is why I didn't do that here. I normally prefer the  'if (bad) return;' style too though, and it seems to be working fine here, so I'd say go for it